Update packages

snakeye opened the pull request.

JackuB commented:
Neat! Thanks a lot! This is a continuation of #168 right?

I'd ask for a single change here before merging: this project is using a [semantic release](https://github.com/semantic-release/semantic-release). It requires a specific commit message format so it can take care of versioning and releasing packages ([see release history](https://github.com/JackuB/apish/releases)).

Could you please update the commit messages (maybe squashing them) to make them look like this: https://www.conventionalcommits.org/ also remove the version from package.json, as that will get generated by semantic release as well.
If you need any help with it, just let me know, I can take care of it.

JackuB commented on src/mock.js:
```diff
@@ -77,14 +77,30 @@ const mock = (refract, options) => {
                 };
-                mocks.push(mockRoute(nockOptions));
+
+              if(!routes.hasOwnProperty(resourceUrl)) {
```
This bit looks misaligned

snakeye replied:
Fixed.

JackuB commented on src/mock.js:
```diff
@@ -77,14 +77,30 @@ const mock = (refract, options) => {
+    const mocks = [];
+    urls.forEach(url => {
+      const resources = routes[url];
+      resources.forEach(resource => {mocks.push(mockRoute(resource))})
```
You could drop curly braces here

```suggestion
      resources.forEach(resource => mocks.push(mockRoute(resource)))
```

snakeye replied:
Fixed.

snakeye commented:
Yes, it's continuation of #168 but I made it as separate PR to be merged afterwards as it is not so critical.

I have fixed the issues and added correct commit messages.

JackuB commented:
:tada: This PR is included in version 2.0.1 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/apish)
- [GitHub release](https://github.com/JackuB/apish/releases/tag/v2.0.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
